src/main/java/com/eprosima/fastcdr/idl/templates/TypesSource.stg

```diff
@@ -519,7 +454,7 @@ $union.scopedname$::$union.name$(
 
```

Comment:
The following union IDL included in the test suite is issuing warnings:

```
union Union_Discriminator_boolean switch (boolean)
{
    case TRUE:
        long first;
    case FALSE:
        long long second;
};
```

```
/home/eprosima/FastDDS/fastdds-docs/src/fastddsgen/build/test/integration/idls/unions/thirdparty/dds-types-test/IDL/unions.cxx: In copy constructor ‘Union_Discriminator_boolean::Union_Discriminator_boolean(const Union_Discriminator_boolean&)’:
    /home/eprosima/FastDDS/fastdds-docs/src/fastddsgen/build/test/integration/idls/unions/thirdparty/dds-types-test/IDL/unions.cxx:6857:12: warning: switch condition has boolean value [-Wswitch-bool]
     6857 |     switch (m__d)
          |            ^
    /home/eprosima/FastDDS/fastdds-docs/src/fastddsgen/build/test/integration/idls/unions/thirdparty/dds-types-test/IDL/unions.cxx: In constructor ‘Union_Discriminator_boolean::Union_Discriminator_boolean(Union_Discriminator_boolean&&)’:
    /home/eprosima/FastDDS/fastdds-docs/src/fastddsgen/build/test/integration/idls/unions/thirdparty/dds-types-test/IDL/unions.cxx:6878:12: warning: switch condition has boolean value [-Wswitch-bool]
     6878 |     switch (m__d)
          |            ^
    /home/eprosima/FastDDS/fastdds-docs/src/fastddsgen/build/test/integration/idls/unions/thirdparty/dds-types-test/IDL/unions.cxx: In member function ‘Union_Discriminator_boolean& Union_Discriminator_boolean::operator=(const Union_Discriminator_boolean&)’:
    /home/eprosima/FastDDS/fastdds-docs/src/fastddsgen/build/test/integration/idls/unions/thirdparty/dds-types-test/IDL/unions.cxx:6899:12: warning: switch condition has boolean value [-Wswitch-bool]
     6899 |     switch (m__d)
          |            ^
    /home/eprosima/FastDDS/fastdds-docs/src/fastddsgen/build/test/integration/idls/unions/thirdparty/dds-types-test/IDL/unions.cxx: In member function ‘Union_Discriminator_boolean& Union_Discriminator_boolean::operator=(Union_Discriminator_boolean&&)’:
    /home/eprosima/FastDDS/fastdds-docs/src/fastddsgen/build/test/integration/idls/unions/thirdparty/dds-types-test/IDL/unions.cxx:6922:12: warning: switch condition has boolean value [-Wswitch-bool]
     6922 |     switch (m__d)
          |            ^
    /home/eprosima/FastDDS/fastdds-docs/src/fastddsgen/build/test/integration/idls/unions/thirdparty/dds-types-test/IDL/unions.cxx: In member function ‘bool Union_Discriminator_boolean::operator==(const Union_Discriminator_boolean&) const’:
    /home/eprosima/FastDDS/fastdds-docs/src/fastddsgen/build/test/integration/idls/unions/thirdparty/dds-types-test/IDL/unions.cxx:6948:12: warning: switch condition has boolean value [-Wswitch-bool]
     6948 |     switch (m__d)
          |            ^
```

Reply:
This comment has been overlooked

Reply:
This warning will be fixed when fixing unions